src/dom/dom.js

```diff
@@ -2221,12 +2221,23 @@ p5.prototype.createCapture = function(...args) {
   let useAudio = true;
   let constraints;
   let callback;
+  let flipped = false;
+
   for (const arg of args) {
     if (arg === p5.prototype.VIDEO) useAudio = false;
     else if (arg === p5.prototype.AUDIO) useVideo = false;
-    else if (typeof arg === 'object') constraints = arg;
-    else if (typeof arg === 'function') callback = arg;
+    else if (typeof arg === 'object') {
+      constraints = arg;
```

Comment:
This gets passed into [`getUserMedia`](https://developer.mozilla.org/en-US/docs/Web/API/MediaDevices/getUserMedia) below. We might want to extract just the bits we need for `constraints` so we don't accidentally send extra info in in the future if we add anything to the settings object. Maybe we could do something like this?
```js
constraints = { video: arg.video, audio: arg.audio };
```